src/desktop/apps/auction_support/client/registration_form.coffee

```diff
@@ -116,7 +118,21 @@ module.exports = class RegistrationForm extends ErrorHandlingForm
     $element.addClass 'is-loading'
     action().finally => $element.removeClass 'is-loading' unless @comboForm
 
+  validateAcceptTerms: (e) ->
+    if @$acceptTerms.prop('checked')
+      @$('.artsy-checkbox').removeClass('error')
+      @enableSubmit()
+      true
+    else
+      @$('.artsy-checkbox').addClass('error')
+      @disableSubmit()
+      false
+  
+  disableSubmit: -> @$submit.addClass('is-disabled')
```

Comment:
You might be able to consolidate these slightly by using `toggleClass` and just having one function switch between adding/removing these classes.